src/mono/wasm/build/WasmApp.targets

```diff
@@ -134,7 +134,26 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_WasmGenerateAppBundle" Condition="'$(WasmGenerateAppBundle)' == 'true'">
+  <Target Name="WasmGenerateRuntimeConfig">
+    <PropertyGroup>
+      <RuntimeConfigFilePath>$(TargetDir)publish\$(AssemblyName).runtimeconfig.json</RuntimeConfigFilePath>
+      <ParsedRuntimeConfigFilePath>$(TargetDir)publish\runtimeconfig.bin</ParsedRuntimeConfigFilePath>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(RunAOTCompilation)' == 'true'">
+      <RuntimeConfigReservedProperties Include="RUNTIME_IDENTIFIER"/>
+      <RuntimeConfigReservedProperties Include="APP_CONTEXT_BASE_DIRECTORY"/>
```

Comment:
I don't understand your question here, in terms of why you would think they are AOT specific. These two properties are the ones set via `monovm_initialize` explicitly in wasm driver.c. So we don't want the user overwrites them.

Reply:
> I don't understand your question here, in terms of why you would think they are AOT specific. 

` <ItemGroup Condition="'$(RunAOTCompilation)' == 'true'">`

I was thinking because of this condition.

Also, we should probably add a comment in `driver.c` with those keys, suggesting that this file should be updated whenever that list of keys changes.

Reply:
Ah I see, it is the condition for these properties. I need to look into it to confirm if the condition is needed. I didn't think too much about it when following the example for android and ios https://github.com/dotnet/runtime/blob/881f3c6c2b9800dfce3c0d2dd1a98771364754e2/eng/testing/tests.mobile.targets#L61

I will add a comment to `driver.c`

Reply:
Removed the AOT condition.